src/main/java/org/folio/circulation/domain/CheckInContext.java

```diff
@@ -94,7 +94,7 @@ public CheckInContext withLoan(Loan loan) {
   public CheckInContext withRequestQueue(RequestQueue requestQueue) {
     Request firstRequest = null;
 
-    if (requestQueue.hasOutstandingFulfillableRequests()) {
+    if (requestQueue.hasOutstandingFulfillableByItemRequests(item)) {
```

Comment:
```suggestion
    if (requestQueue.hasOutstandingRequestsFulfillableByItem(item)) {
```

Reply:
yes, I would name it as you recommended, but the method name suggested in the requirement (CIRC-1421 Jira ticket)

Reply:
I updated the ticket

Reply:
done